src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
I should probably let @etpinard handle this but I think you want both `_isLinkedToArray` and `_arrayAttrRegexps` - the latter of which won't get used yet but when we extend the array editing stuff I did in `relayout` to `restyle` it will be important.

see https://github.com/plotly/plotly.js/blob/master/src/components/updatemenus/attributes.js#L17-L52